monitoring/snippets/v3/cloud-client/snippets_test.py

```diff
@@ -52,72 +52,56 @@ def write():
     yield
 
 
-def test_get_delete_metric_descriptor(capsys, custom_metric_descriptor):
+def test_get_delete_metric_descriptor(capsys, custom_metric_descriptor) -> None:
     try:
 
         @backoff.on_exception(backoff.expo, (AssertionError, NotFound), max_time=60)
         def eventually_consistent_test():
-            snippets.get_metric_descriptor(custom_metric_descriptor)
-            out, _ = capsys.readouterr()
-            assert "DOUBLE" in out
+            result = snippets.get_metric_descriptor(custom_metric_descriptor)
+            assert result.value_type == 3  # (3 == MetricDescriptor.ValueType.DOUBLE)
 
         eventually_consistent_test()
     finally:
         snippets.delete_metric_descriptor(custom_metric_descriptor)
-        out, _ = capsys.readouterr()
-    assert "Deleted metric" in out
 
 
 @backoff.on_exception(backoff.expo, (ServiceUnavailable), max_tries=3)
-def test_list_metric_descriptors(capsys):
-    snippets.list_metric_descriptors(PROJECT_ID)
-    out, _ = capsys.readouterr()
-    assert "logging.googleapis.com/byte_count" in out
+def test_list_metric_descriptors(capsys) -> None:
+    result = snippets.list_metric_descriptors(PROJECT_ID)
+    assert any(item.type == "logging.googleapis.com/byte_count" for item in result)
 
 
 @backoff.on_exception(backoff.expo, (ServiceUnavailable), max_tries=3)
-def test_list_resources(capsys):
-    snippets.list_monitored_resources(PROJECT_ID)
-    out, _ = capsys.readouterr()
-    assert "pubsub_topic" in out
+def test_list_resources(capsys) -> None:
+    result = snippets.list_monitored_resources(PROJECT_ID)
+    assert any(item.type == "pubsub_topic" for item in result)
 
 
 @backoff.on_exception(backoff.expo, (ServiceUnavailable), max_tries=3)
-def test_get_resources(capsys):
-    snippets.get_monitored_resource_descriptor(PROJECT_ID, "pubsub_topic")
-    out, _ = capsys.readouterr()
-    assert "A topic in Google Cloud Pub/Sub" in out
+def test_get_resources(capsys) -> None:
+    result = snippets.get_monitored_resource_descriptor(PROJECT_ID, "pubsub_topic")
+    assert result.display_name == "Cloud Pub/Sub Topic"
 
 
 @backoff.on_exception(backoff.expo, (ServiceUnavailable), max_tries=3)
-def test_list_time_series(capsys, write_time_series):
-    snippets.list_time_series(PROJECT_ID)
-    out, _ = capsys.readouterr()
-    assert "gce_instance" in out
+def test_list_time_series(capsys, write_time_series) -> None:
```

Comment:
ditto: We probably don't need `capsys` anymore.

Reply:
done